relay/monitor.go

```diff
@@ -223,3 +221,24 @@ func (m *Monitor) waitForSetStatusProposal(ctx context.Context, ch chan state) {
 		ch <- stop
 	}
 }
+
+// helpers
+
+func (m *Monitor) wasQuorumReached(quorum *big.Int, count *big.Int) bool {
+	return quorum.Cmp(count) == 0 || quorum.Cmp(count) == -1
```

Comment:
What happens if a node sends 2 signatures (due to a bug). 

Could have expressed this:
```GO
return quorum.Cmp(count) <= 0
```

Reply:
Good question, signatures are stored in a hash with the peerId as key, https://github.com/ElrondNetwork/elrond-eth-bridge/blob/050ffccc00cd86218dc7d8dfe43fef16260e14f1/relay/relay.go#L339

Reply:
so no double signs are possible. Good!